share/spice/reddit_user_search/content.handlebars

```diff
@@ -0,0 +1,7 @@
+<h5>
+    {{name}}
+</h5>
+<div>{{reddit_user_search_addCommas link_karma}} link karma</div>
```

Comment:
I would suggest you use our `record` sub template for these. It'll put the data into a nice key-value table.

>Details: https://duck.co/duckduckhack/spice_templates_overview#record-template

The end result should look like the [Whois Spice](https://duckduckgo.com/?q=whois+awesomewebsite.com&ia=whois). Here's the code for reference: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/whois/whois.js